demucs/apply.py

```diff
@@ -323,17 +319,17 @@ def apply_model(model: tp.Union[BagOfModels, Model],
             try:
                 callback(_replace_dict(callback_arg, ("state", "start")))  # type: ignore
             except KeyboardInterrupt:
-                raise
+                return None
             except Exception:
-                pass
+                traceback.print_exc(file=sys.stderr)
         with th.no_grad():
             out = model(padded_mix)
         with lock:
             try:
                 callback(_replace_dict(callback_arg, ("state", "end")))  # type: ignore
             except KeyboardInterrupt:
-                raise
+                return None
             except Exception:
-                pass
+                traceback.print_exc(file=sys.stderr)
```

Comment:
I think you shouldn't catch the exception and proceed silently, nor print the traceback, you should just let the exception be raised, e.g. just remove the `except Exception` bit.